### ISSUE TITLE
0.0.42 Rollup cannot find resource main.css

### ISSUE THREAD

**Issue by pmurpmur** [driftyco/ionic-app-scripts#343]
#### Short description of the problem:
Rollup doesn't work in 0.0.42, whereas it did work in earlier versions.

#### What behavior are you expecting?
To be able to use Rollup over Webpack.

**Steps to reproduce:**
1. Create new Ionic project
2. Add `"ionic_bundler": "rollup"` to `"config"` in `package.json`
3. `ionic serve`

Results in:
```
> ionic-hello-world@ ionic:serve /Users/Development/project/test
> ionic-app-scripts serve "[object Object]"

[20:38:17]  ionic-app-scripts 0.0.42 
[20:38:17]  watch started ... 
[20:38:17]  build dev started ... 
[20:38:17]  clean started ... 
[20:38:17]  clean finished in 4 ms 
[20:38:17]  copy started ... 
[20:38:17]  transpile started ... 
[20:38:19]  transpile finished in 1.85 s 
[20:38:19]  rollup started ... 
[20:38:19]  copy finished in 1.88 s 
[20:38:21]  rollup: Conflicting namespaces:  ...
[20:38:25]  rollup finished in 5.45 s 
[20:38:25]  sass started ... 
[20:38:25]  build dev failed: Path must be a string without null bytes 
[20:38:25]  watch ready in 7.33 s 
[20:38:25]  dev server running: http://localhost:8100/ 
```

First off, in the second line it says `ionic-app-scripts serve "[object Object]"` (this happens with webpack as well). Interestingly, this problem can be solved by using `npm run ionic:serve "PATH_TO_YOUR_PROJECT"` instead of `ionic serve`.

Secondly, above where it says `rollup: Conflicting namespaces:  ...` I get warnings (which I omitted) identical to this issue: https://github.com/driftyco/ionic-app-scripts/issues/327

Lastly, I get the error `build dev failed: Path must be a string without null bytes` which causes the app to be launched in browser but `main.css` responds with a 404 causing styling issues.

**Which @ionic/app-scripts version are you using?**
"@ionic/app-scripts": "0.0.42"

Cordova CLI: 6.4.0 
Ionic Framework Version: 2.0.0-rc.2
Ionic CLI Version: 2.1.7
Ionic App Lib Version: 2.1.4
Ionic App Scripts Version: 0.0.42
ios-deploy version: 1.9.0 
ios-sim version: 5.0.10 
OS: macOS Sierra
Node Version: v7.0.0
Xcode version: Xcode 8.1 Build version 8B62

**Comment by danbucholtz** [comment]
This will be fixed in `0.0.43` today.

Thanks,
Dan

**Comment by pmurpmur** [comment]
The main issue has been fixed in `0.0.43`. Thanks.
